test/erlfmt_format_SUITE.erl

```diff
@@ -1710,6 +1710,30 @@ receive_expression(Config) when is_list(Config) ->
         "    % two\n"
         "    2 -> two\n"
         "end\n"
+    ),
+    % ?assertSame(
+    %     "receive\n"
+    %     "after\n"
+    %     "    % foo\n"
+    %     "    0 -> ok\n"
+    %     "end\n"
+    % ),
+    ?assertFormat(
+        "receive\n"
+        "    1 -> ok\n"
+        "    %% after receive\n"
+        "after\n"
+        "    0 -> ok\n"
+        "    %% after after for receive\n"
+        "end\n",
+        "receive\n"
+        "    1 -> ok\n"
+        "%% after receive\n"
+        "\n"
+        "after 0 ->\n"
+        "    ok\n"
+        "    %% after after for receive\n"
+        "end\n"
```

Comment:
Let's maybe keep the receive changes completely out of this. What do you think?

Reply:
I think it is easy to modify when we change the behaviour, rather than rewrite the test again once we start working on this again.